docs/translations/translations-aa.json

```diff
@@ -130,7 +130,7 @@
     "/components/dialogs": "crwdns106397:0crwdne106397:0",
     "/components/dividers": "crwdns106399:0crwdne106399:0",
     "/components/drawers": "crwdns106401:0crwdne106401:0",
-    "/components/expansion-panels": "crwdns106403:0crwdne106403:0",
+    "/components/accordions": "crwdns106403:0crwdne106403:0",
```

Comment:
Should I track somewhere the translation changes needed? Should I completely revert these changes

Reply:
/cc @mbrookes

Reply:
It might be interesting to keep the changes in GitHub, so that when we synchronize, we can assert the quality it.

Reply:
We'll need to coordinate the renaming of the markdown files in Crowdin between after this is merged, and before staging is updated. Keeping the manual changes should be harmless. We can get rid of the `aa` files at some point.